test/Microsoft.NET.Build.Tests/GivenThatWeWantToVerifyPCLProjectReferenceCompat.cs

```diff
@@ -0,0 +1,115 @@
+﻿using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+using static Microsoft.NET.TestFramework.Commands.MSBuildTest;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Xunit;
+using FluentAssertions;
+using System.Runtime.InteropServices;
+using System.Linq;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.Build.Tests
+{
+    public class GivenThatWeWantToVerifyPCLProjectReferenceCompat : SdkTest
+    {
+        public GivenThatWeWantToVerifyPCLProjectReferenceCompat(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        [Theory]
+        [InlineData("netstandard1.1", "Profile7", "v4.5", true, true)]
+        [InlineData("netstandard1.0", "Profile31", "v4.6", true, true)]
+        [InlineData("netstandard1.2", "Profile32", "v4.6", true, true)]
+        [InlineData("netstandard1.2", "Profile44", "v4.6", true, true)]
+        [InlineData("netstandard1.0", "Profile49", "v4.5", true, true)]
+        [InlineData("netstandard1.0", "Profile78", "v4.5", true, true)]
+        [InlineData("netstandard1.0", "Profile84", "v4.6", true, true)]
+        [InlineData("netstandard1.1", "Profile111", "v4.5", true, true)]
+        [InlineData("netstandard1.2", "Profile151", "v4.6", true, true)]
+        [InlineData("netstandard1.0", "Profile157", "v4.6", true, true)]
+        [InlineData("netstandard1.0", "Profile259", "v4.5", true, true)]
```

Comment:
Not sure what the bools are for if they are always true,true.

We should probably also add cases that we expect to fail.